pipeline-steps/templates/steps/acr-login.yaml

```diff
@@ -0,0 +1,24 @@
+parameters:
+  azureSubscriptionEndpoint: ''
+  aksResourceGroup: ''
+  aksCluster: ''
+  azureContainerRegistry: ''
+
+steps:
+  - task: AzureCLI@1
```

Comment:
I think worth to split this into a separate task, It's already used on helm-install task